OpenTabletDriver.Desktop/Diagnostics/LinuxEtcOSRelease.cs

```diff
@@ -0,0 +1,64 @@
+using System;
+using System.IO;
+using System.Collections.Generic;
+
+namespace OpenTabletDriver.Desktop.Diagnostics
+{
+    public class LinuxEtcOSRelease
```

Comment:
```suggestion
    public class LinuxOSRelease
```

`Etc` irrelevant IMO.

Reply:
Should've added more context. I say irrelevant because `etc/` is not the only place `os-release` can exist, per the manpage. (This is also something that might be good to consider for the parser logic, though IDK how many distros require handling this edge case)